joern-cli/frontends/gosrc2cpg/src/main/scala/io/joern/gosrc2cpg/utils/AstGenRunner.scala

```diff
@@ -134,13 +134,15 @@ class AstGenRunner(config: Config) {
     }
   }
 
-  private def runAstGenNative(in: String, out: File): Try[Seq[String]] =
-    ExternalCommand.run(s"$astGenCommand -out ${out.toString()} $in", ".")
+  private def runAstGenNative(in: String, out: File, exclude: String): Try[Seq[String]] = {
+    val excludeCommand = if (exclude.isEmpty) "\"\"" else s"\"$exclude\""
```

Comment:
@ankit-privado Can you try following? 

```
val excludeCommand = if (exclude.isEmpty) "" else s"-exclude $exclude"
ExternalCommand.run(s"$astGenCommand $excludeCommand -out ${out.toString()} $in", ".")

```

Reply:
initially I was also doing this, but this is command line string. if we do this then command look like `..-exclude -out..` but we need `..-exclude "" -out..`

@pandurangpatil

Reply:
What I am suggesting should generate `.. -out ..`

Reply:
If `goastgen` always expects a value to be passed for `-exclude`, then I suppose we need to fix that on `goastgen`

Reply:
okay okay.